go/vt/vtgate/vtgate.go

```diff
@@ -26,6 +26,8 @@ import (
 	"time"
 
 	log "github.com/golang/glog"
+	prom "github.com/prometheus/client_golang/prometheus"
```

Comment:
I think if we do the abstractions right, there should be no references to prom anywhere outside the `stats` package.